libpod/stats_linux.go

```diff
@@ -137,3 +138,15 @@ func calculateBlockIO(stats *runccgroup.Stats) (read uint64, write uint64) {
 	}
 	return
 }
+
+func getOnlineCPUs(container *Container) (int, error) {
+	ctrPID, err := container.PID()
```

Comment:
I think you need handling for ctrPID being returned as 0 - we can do that if the container is stopped.

Reply:
Probably sufficient to just return ErrCtrStopped in that case.